modules/ipi-install-replacing-a-bare-metal-control-plane-node.adoc

```diff
@@ -60,30 +62,35 @@ $ cat <<EOF | oc apply -f -
 apiVersion: v1
 kind: Secret
 metadata:
-  name: kni1-master-0-bmc-secret
+  name: control-plane-<num>-bmc-secret <1> 
   namespace: openshift-machine-api
 data:
-  password: <username>
-  username: <password>
+  password: <base64-of-uid> <2>
```

Comment:
Please use underscores within replaceable values (e.g., `<base64_of_uid>`) uniformly throughout the PR, per [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output).

Reply:
I updated the replaceable values with underscores, thanks!